_includes/news-section.html

```diff
@@ -0,0 +1,11 @@
+<article class="c-latest-news">
+<div class="c-latest-news__title-container">
+  <h1 class="c-latest-news__title">Latest News</h1>
+  <a href="#" class="c-latest-news__link c-latest-news__link--border">View All News</a>
+</div>
```

Comment:
Let's add a line break here to make the code easier to read.

Reply:
I did a beautify on it. Should be good to go :)